Train.py

```diff
@@ -62,6 +62,10 @@ def run_net(data_index):
 
                 # Logging Loss
                 epoch_train_loss.add(data.train_index.ctr, batch.loss.data[0])
+                epoch_train_loss.export_csv(
+                    'logs/epoch%02d_train_loss.csv' %
```

Comment:
Is it going to be a good idea to re-export after every batch? Maybe it should be changed to only writing one line at a time rather than overwriting the whole file.